src/.babelrc.js

```diff
@@ -0,0 +1,21 @@
+const pkg = require('../package.json')
+const version = pkg.version
+
+module.exports = {
+    plugins: [
+        [
+            'minify-replace',
+            {
+                replacements: [
+                    {
+                        identifierName: '__VERSION__',
+                        replacement: {
+                            type: 'stringLiteral',
+                            value: version,
```

Comment:
Since we use the platform (which uses CRA under the hood) we can't customize anything about the webpack config, so we have to do any transformations at the Babel pre-compile step

Reply:
Right, makes a lot of sense! Totally slipped my attention that this was converted to a platform app - nice!

Reply:
You're so out of touch!  (it happened on Tuesday ;-) #110)